src/platform/site-wide/mega-menu/containers/Main.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import { createSelector } from 'reselect';
 import { connect } from 'react-redux';
-import defaultLinkData from '../mega-menu-link-data.json';
+import defaultLinkData from '../../../../../.cache/megamenu.json';
```

Comment:
I don't like this, but seeing as how it's generated out of content, I think the alternative would be to use HTTP to get the MegaMenu config, and at the moment I don't think we can justify the render delay that would come out of that.